packages/modal-ui-js/src/lib/components/GetAWallet.ts

```diff
@@ -64,7 +64,14 @@ export async function renderGetAWallet() {
     </div>
   `;
 
-  const filteredModules = modalState.modules.slice(0, 3);
+  const filterByID = (item: { type: string }) => {
+    if (item.type !== "bridge" && item.type !== "hardware") {
+      return true;
+    }
+    return false;
+  };
```

Comment:
Just a minor comment from me, we could probably shorten/simplify this method:

```ts
const filterByID = (item: { type: string }) => {
   return item.type !== "bridge" && item.type !== "hardware";
};
```

I think we could rename the `filterByID` to `filterByType` because we're filtering wallets by wallet type and not by id.